index.js

```diff
@@ -1 +1 @@
rget.value))}}),(0,r.h)("div",{class:"current-time"},[e.stateIdx])]),(0,r.h)("div",{class:"controls"},[e.isTimeTravelling&&e.stateIdx!==e.states.length-1&&(0,r.h)("div",{class:"button detach",onclick:t.detachResume,title:"Resume interactivity, but from the current state"},["Detach"]),(0,r.h)("div",{key:"playback",class:["button playback",e.isTimeTravelling?"resume":"pause"].join(" "),onclick:function(){return e.isTimeTravelling?t.resume():t.timeTravelTo(e.states.length-1)},title:"Toggle debug mode."})]),(0,r.h)("div",{id:"hyperapp-debugger-explorer"},[e.states.length>0&&o.view(e.states[e.stateIdx],t.explorer)]),(0,r.h)("div",{class:"external"},[(0,r.h)("a",{href:"#",onclick:t.export},["Export"])," / ",(0,r.h)("label",{for:"hyperapp-debug-import"},["Import"]),(0,r.h)("input",{id:"hyperapp-debug-import",style:{outline:"none",width:0,height:0},type:"file",onchange:t.import})])])}(e,t)])}},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0}),t.view=t.actions=t.initialState=void 0;var r,o=n(0),a=(r=n(9))&&r.__esModule?r:{default:r};function i(e){return(i="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e})(e)}function s(){return(s=Object.assign||function(e){for(var t=1;t<arguments.length;t++){var n=arguments[t];for(var r in n)Object.prototype.hasOwnProperty.call(n,r)&&(e[r]=n[r])}return e}).apply(this,arguments)}t.initialState={exposeId:1};var c={exposeVariable:function(e){return function(t){var n="$debug".concat(t.exposeId);return console.log("[HyperDebug]","window.".concat(n)),window[n]=e,console.log(window[n]),{exposeId:t.exposeId+1}}}};t.actions=c;t.view=function(e,t){return(0,o.h)(u,s({value:e},t))};var u=function e(t){var n=t.exposeVariable,r=t.name,s=t.value,c=function(){return void 0!==i(r)?(0,o.h)("a",{class:"object-key",href:"#",onclick:function(e){e.preventDefault(),n(s)}},[r]):null},u=(0,a.default)(s)?"{}":Array.isArray(s)?"[]":null,p=Object.keys(s),l="";if(Array.isArray(s)&&s.length>10){s.length;p=p.slice(-10),l="..."}if(u)return(0,o.h)("div",{},[c()," ".concat(u[0]),(0,o.h)("div",{class:"object"},[l,p.filter(function(e){return!e.startsWith("$")}).map(function(t){return(0,o.h)(e,{key:t,exposeVariable:n,name:t,value:s[t]})})]),u[1]]);var f=JSON.stringify(s);return(0,o.h)("div",{},[c(),f.length>15?"...":"",f.slice(-15)])}},function(e,t){var n="[object Object]";var r,o,a=Function.prototype,i=Object.prototype,s=a.toString,c=i.hasOwnProperty,u=s.call(Object),p=i.toString,l=(r=Object.getPrototypeOf,o=Object,function(e){return r(o(e))});e.exports=function(e){if(!function(e){return!!e&&"object"==typeof e}(e)||p.call(e)!=n||function(e){var t=!1;if(null!=e&&"function"!=typeof e.toString)try{t=!!(e+"")}catch(e){}return t}(e))return!1;var t=l(e);if(null===t)return!0;var r=c.call(t,"constructor")&&t.constructor;return"function"==typeof r&&r instanceof r&&s.call(r)==u}}]).debug});
```

Comment:
Is this the minified output being checked in to source control? 😕

Reply:
This can be fixed 💯